x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/flyout_wrapper.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiToolTip,
+  EuiButton,
+  EuiLink,
+  EuiBetaBadge,
+} from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { FlyoutWrapperProps } from './types';
+
+export const FlyoutWrapper = ({
+  children,
+  isInlineFlyoutVisible,
+  isScrollable,
+  displayFlyoutHeader,
+  language,
+  onCancel,
+  navigateToLensEditor,
+  onApply,
+}: FlyoutWrapperProps) => {
+  return (
+    <>
+      {isInlineFlyoutVisible && displayFlyoutHeader && (
+        <EuiFlyoutHeader
+          hasBorder
+          css={css`
+            pointer-events: auto;
+          `}
+          data-test-subj="editFlyoutHeader"
+        >
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="center" responsive={false}>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup alignItems="center" gutterSize="xs">
+                <EuiFlexItem grow={false}>
+                  <EuiTitle size="xs">
+                    <h2 id="Edit visualization">
```

Comment:
why do we add id that is a white-spaced string? Any reason for it?

Reply:
No reason I just took an example from EUI, I will remove it.